src/workerd/jsg/dom-exception.h

```diff
@@ -112,6 +113,15 @@ class DOMException: public Object {
     tracker.trackField("errorForStack", errorForStack);
   }
 
+  // The value is taken from worker-interface.capnp, which we can't depend
+  // on directly here so we have to set it manually.
+  static const uint SERIALIZATION_TAG = 0x7;
```

Comment:
Nitpick: I don't think there's any reason for this to be hex -- the `@N` tags in Cap'n Proto are decimal.

```suggestion
  static const uint SERIALIZATION_TAG = 7;
```